packages/atomic/src/components/commerce/product-template-components/atomic-product-text/e2e/atomic-product-text.e2e.ts

```diff
@@ -0,0 +1,95 @@
+import {test, expect} from './fixture';
+
+test.describe('default', async () => {
+  test.beforeEach(async ({productText}) => {
+    await productText.load();
+    await productText.hydrated.first().waitFor();
+  });
+
+  test('should be accessible', async ({makeAxeBuilder}) => {
+    const accessibilityResults = await makeAxeBuilder().analyze();
+    expect(accessibilityResults.violations.length).toEqual(0);
+  });
+
+  test.describe('when field has no value and default is set', async () => {
+    test('should render default text', async ({productText}) => {
+      await productText.load({
+        args: {field: 'nonexistentField', default: 'Default Text'},
+      });
+      await productText.hydrated.first().waitFor();
+
+      expect(productText.textContent.first()).toContainText('Default Text');
+    });
+  });
+});
+
+test.describe('when using a field that supports highlights', async () => {
+  const fields = ['excerpt', 'ec_name'];
+
+  fields.forEach((field) => {
+    test.describe(`when displaying the ${field}`, async () => {
```

Comment:
I find it a bit odd that the highlighting is done in the `atomic-product-text component`, as it implies that the `shouldHighlight` prop is completely useless unless the specified `field` is `excerpt` or `ec_name`... 

This is really a comment (not a request for change). I simply wonder if this is the right design decision.

Reply:
I think the idea is we could expand the list of compatible fields which have highlights returned in the request at a later time